### PR TITLE
Add experiments alias for DebugOptions

### PR DESCRIPTION
**Pull request by vikkyrk**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---
To keep parity with Java SDK.

### CODE REVIEW

**Comment by vikkyrk** [comment]
R: @aaltay
CC: @lukecwik

**Comment by aaltay** [comment]
LGTM

File: sdks/python/apache_beam/utils/options.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def _add_argparse_args(cls, parser):
                         default=None,
                         help='Debug file to write the workflow specification.')
     parser.add_argument(
-        '--experiment',
+        '--experiment', "--experiments",
```
Comment (lukecwik) [inline sdks/python/apache_beam/utils/options.py]: Going to swap " to ' for style during merge.

**Comment by lukecwik** [comment]
Merged with:
https://github.com/apache/incubator-beam/commit/c8a8bb21eac7d365159dabd3a70e5cc5cd0c5d28

**Pull request closed without merging**